src/main/java/de/hysky/skyblocker/skyblock/item/SkyblockInventoryScreen.java

```diff
@@ -161,6 +162,7 @@ protected void drawMouseoverTooltip(DrawContext context, int x, int y) {
             if (isPointWithinBounds(equipmentSlot.x, equipmentSlot.y, 16, 16, x, y) && equipmentSlot.hasStack()) {
                 ItemStack itemStack = equipmentSlot.getStack();
                 context.drawTooltip(this.textRenderer, this.getTooltipFromItem(itemStack), itemStack.getTooltipData(), x, y);
+				FocusedItemProvider.setFocusedItem(itemStack);
```

Comment:
Just noticed I forgot to clear the focused item for the inventory screen, will fix later.